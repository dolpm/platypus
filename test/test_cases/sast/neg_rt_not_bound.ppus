pipe test |> [] |> [x: int] |> int {
  |> x;
}

pipe main |> [] |> [] |> unit {
  test <| [1];
  |> ();
}