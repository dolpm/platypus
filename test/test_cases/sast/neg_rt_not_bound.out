Fatal error: exception Failure("If the returned value from the call to isn't a borrow or unit, it must be assigned to an owner.")
